### PR TITLE
Validation: Fix multiple issues 

### PR DESCRIPTION
**Pull request by annmariehitchcock**
This PR addresses the following issues: 
image-alt-text-duplicated
description-missing

 

Fixed #32 issues across #26 files.

### CODE REVIEW

File: dotnet-desktop-guide/framework/wpf/data/data-templating-overview.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ The WPF data templating model provides you with great flexibility to define the
 
  Now our <xref:System.Windows.Controls.ListBox> looks like the following:
 
- ![Data templating sample screenshot](./media/datatemplatingintro-fig3.png "DataTemplatingIntro_fig3")
+ ![Screenshot of the Introduction to Data Templating Sample window showing the My Task List ListBox displaying the tasks as TextBlock elements. The task Email, Email clients, Priority 1 is highlighted in blue.](./media/datatemplatingintro-fig3.png "DataTemplatingIntro_fig3")
```
Comment (brittmsantos) [inline dotnet-desktop-guide/framework/wpf/data/data-templating-overview.md:64]: Maximum length for alt text descriptions is 150 characters.

**Comment by annmariehitchcock** [comment]
Global effort to fix validation errors

@adegeo

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/wpf/data/attribute-xelement-dynamic-property.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Attribute (XElement dynamic property)
+description: Learn how the attribute (XElement dynamic property) uses an indexer to retrieve the attribute instance that corresponds to the specified expanded name.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/data/attribute-xelement-dynamic-property.md:3]:
```suggestion
description: Learn how the XElement.Attribute property uses an indexer to retrieve the attribute instance that corresponds to the specified expanded name.
```

File: dotnet-desktop-guide/framework/wpf/data/descendants-xelement-dynamic-property.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Descendants (XElement dynamic property)
+description: Learn how the descendants (XElement dynamic property) retrieves all the descendant elements of the current element that match the specified expanded name.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/data/descendants-xelement-dynamic-property.md:3]:
```suggestion
description: Learn how the XElement.Descendants property retrieves all the descendant elements of the current element that match the specified expanded name.
```

File: dotnet-desktop-guide/framework/wpf/data/element-xelement-dynamic-property.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Element (XElement dynamic property)
+description: Learn how the element (XElement dynamic property) uses an indexer to retrieve the child element instance that corresponds to the specified expanded name.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/data/element-xelement-dynamic-property.md:3]:
```suggestion
description: Learn how the XElement.Element property uses an indexer to retrieve the child element instance that corresponds to the specified expanded name.
```

File: dotnet-desktop-guide/framework/wpf/data/elements-xelement-dynamic-property.md [hunk 5]
```diff
@@ -1,5 +1,6 @@
 ---
 title: Elements (XElement dynamic property)
+description: Learn how the element (XElement dynamic property) uses an indexer to retrieve the child elements of the current element that match the specified expanded name.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/data/elements-xelement-dynamic-property.md:3]:
```suggestion
description: Learn how the XElement.Elements property uses an indexer to retrieve the child elements of the current element that match the specified expanded name.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**